libmei/tools/parseschema2.py

```diff
@@ -56,7 +56,7 @@ def main(configure: dict) -> bool:
     p = ArgumentParser(usage='%(prog)s [-c config path] [compiled odd path]')
 
     p.add_argument("compiled", help="A compiled ODD file", type=Path)
-    p.add_argument("-c", "--config", help="Path to a config file", type=Path)
+    p.add_argument("-c", "--config", help="Path to a config file", required=True, type=Path)
```

Comment:
The https://github.com/rism-digital/verovio/blob/develop/libmei/README.md needs to be adjusted accordingly

Reply:
We can probably also make the config argument optional and use a default path when it is left out. How would you prefer? Anyway the script should not crash.

Reply:
I did have a default path in there before, but I think it got missed when I did a refactoring. I can add that back in.